third_party/txt/src/minikin/FontCollection.cpp

```diff
@@ -299,12 +306,20 @@ const std::shared_ptr<FontFamily>& FontCollection::getFamilyForChar(
     uint32_t langListId,
     int variant) const {
   if (ch >= mMaxChar) {
+    for (size_t i = 0; i < sFallbackFamilies.size(); i++) {
```

Comment:
(optional) Since this is identical to the snippet below, consider using helper function to help keep these two sections in sync.

Reply:
Also, since it would be helpful to have a short comment explaining why this is necessary.